backend/src/nodes/nodes/utility/convert_number.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import annotations
+
+from ...node_base import NodeBase
+from ...node_factory import NodeFactory
+from ...properties.inputs import NumberInput, TextInput
+from ...properties.outputs import NumberOutput
+from . import category as UtilityCategory
+
+
+@NodeFactory.register("chainner:utility:convert_number")
+class ConvertNumberNode(NodeBase):
+    def __init__(self):
+        super().__init__()
+        self.description = "Converts a text to base-10."
+        self.inputs = [
+            TextInput("Text", min_length=0),
+            NumberInput("Base", default=10, minimum=2, maximum=36),
+        ]
+        self.outputs = [
+            NumberOutput("Value", output_type="int"),
```

Comment:
I just added support for parsing numbers in the type system, so we will be able to use `number::parseInt` after #1629 is merged, and you rebased your PR branch.

```suggestion
            NumberOutput(
                "Value",
                output_type="int & number::parseInt(Input0, Input1)",
            ).with_never_reason("The given text cannot be parsed into a number."),
```

Reply:
Rebased and updated. Along with renaming to Parse Number.